requirements.txt

```diff
@@ -13,16 +16,25 @@ django-cors-middleware==1.3.1
 django-extensions==2.1.4
 django-heroku==0.3.1
 django-nose==1.4.6
+django-rest-swagger==2.2.0
```

Comment:
@VivianDoreen You have this here but you are not using it. Please ensure everything added to the `requirements.txt` is being used in the application.

Reply:
I have made the necessary changes